src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -80,9 +84,54 @@ void shouldModifyUser() {
         );
     }
 
+    @Test
+    void getUserByEmail() {
+        createUser();
+
+        Response response = buildRequestWithExternalGlobalAccessAuth()
+            .baseUri(getUri(ADMIN_USERS))
+            .queryParam(COURTHOUSE_ID, 21)
+            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
+            .get()
+            .thenReturn();
+
+        assertEquals(200, response.getStatusCode());
+
+    }
+
+    @Test
+    void getUserByEmailNoEmailProvided() {
+        createUser();
+
+        Response response = buildRequestWithExternalGlobalAccessAuth()
+            .baseUri(getUri(ADMIN_USERS))
+            .queryParam(COURTHOUSE_ID, 21)
+            .header(EMAIL_ADDRESS, "james.smith@hmcts.com")
+            .get()
+            .thenReturn();
+
+        assertEquals(200, response.getStatusCode());
+
+    }
+
+    @Test
+    void getUserByEmailNotAdminUser() {
+        createUser();
+
+        Response response = buildRequestWithExternalAuth()
+            .baseUri(getUri(ADMIN_USERS))
+            .queryParam(COURTHOUSE_ID, 21)
+            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
+            .get()
+            .thenReturn();
+
+        assertEquals(403, response.getStatusCode());
```

Comment:
Check response

Reply:
assertion added to check user email does not exist